readthedocs/rtd_tests/fixtures/spec/v2/schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Schema for Read the Docs configuration file",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "basics": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+	      "version": "enum('2')",
+	      "formats": "any(list(enum('htmlzip', 'pdf', 'epub')), enum('all'), required=False)",
```

Comment:
Believe this isn't the way of doing a list of option is json schema, and all others too (boolean values, enums, etc)

Reply:
ok I will look into it

Reply:
@stsewd Should the it go like`"properties": {
              "version": "enum['2']",
              "formats": "any(array[enum['htmlzip', 'pdf', 'epub']], enum['all'], required=False)",
              "conda": "include('conda', required=False)",
              "build": "include('build', required=False)",
`

Reply:
No, please refer to https://json-schema.org/learn/getting-started-step-by-step.html if you are not familiar with json schema

Reply:
@stsewd I looked into it 
does the code go like this
  "properties": {
              "version":{
                      "description":"The version of the spec to be use",
                      "type":"string",
                      "enum":["2"]
              },